bundles/org.openhab.binding.lgthinq/src/main/java/org/openhab/binding/lgthinq/internal/LGThinQBindingConstants.java

```diff
@@ -0,0 +1,145 @@
+    // === UserData Directory and File Format
+    public static String THINQ_USER_DATA_FOLDER = OpenHAB.getUserDataFolder() + File.separator + "thinq";
+    public static String THINQ_CONNECTION_DATA_FILE = THINQ_USER_DATA_FOLDER + File.separator + "thinqbridge-%s.json";
+    public static String BASE_CAP_CONFIG_DATA_FILE = THINQ_USER_DATA_FOLDER + File.separator + "thinq-%s-cap.json";
+
+    // ====================================================
+
+    /**
+     * ============ Air Conditioner Channels & Constant Definition =============
+     */
+    public static final String CHANNEL_AC_AIR_CLEAN_ID = "air-clean";
+    public static final String CHANNEL_AC_AIR_WATER_SWITCH_ID = "air-water-switch";
+    public static final String CHANNEL_AC_AUTO_DRY_ID = "auto-dry";
+    public static final String CHANNEL_AC_COOL_JET_ID = "cool-jet";
+    public static final String CHANNEL_AC_CURRENT_POWER_ID = "current-power";
+    public static final String CHANNEL_AC_CURRENT_TEMP_ID = "current-temperature";
+    public static final String CHANNEL_AC_ENERGY_SAVING_ID = "energy-saving";
+    public static final String CHANNEL_AC_FAN_SPEED_ID = "fan-speed";
+    public static final String CHANNEL_AC_MAX_TEMP_ID = "max_temperature";
+    public static final String CHANNEL_AC_MIN_TEMP_ID = "min_temperature";
+    public static final String CHANNEL_AC_MOD_OP_ID = "op-mode";
+    public static final String CHANNEL_AC_POWER_ID = "power";
+    public static final String CHANNEL_AC_REMAINING_FILTER_ID = "remaining-filter";
+    public static final String CHANNEL_AC_STEP_LEFT_RIGHT_ID = "fan-step-left-right";
+    public static final String CHANNEL_AC_STEP_UP_DOWN_ID = "fan-step-up-down";
+    public static final String CHANNEL_AC_TARGET_TEMP_ID = "target-temperature";
+
+    /**
+     * ============ Refrigerator's Channels & Constant Definition =============
+     */
+    public static final String CHANNEL_RE_ACTIVE_SAVING = "fr-active-saving";
+    public static final String CHANNEL_RE_DOOR_OPEN = "some-door-open";
+    public static final String CHANNEL_RE_EXPRESS_COOL_MODE = "fr-express-cool-mode";
+    public static final String CHANNEL_RE_EXPRESS_FREEZE_MODE = "fr-express-mode";
+    public static final String CHANNEL_RE_FREEZER_TEMP_ID = "freezer-temperature";
+    public static final String CHANNEL_RE_FRESH_AIR_FILTER = "fr-fresh-air-filter";
+    public static final String CHANNEL_RE_FRIDGE_TEMP_ID = "fridge-temperature";
+    public static final String CHANNEL_RE_ICE_PLUS = "fr-ice-plus";
+    public static final String CHANNEL_RE_REF_TEMP_UNIT = "temp-unit";
+    public static final String CHANNEL_RE_SMART_SAVING_MODE_V2 = "fr-smart-saving-mode";
+    public static final String CHANNEL_RE_SMART_SAVING_SWITCH_V1 = "fr-smart-saving-switch";
+    public static final String CHANNEL_RE_VACATION_MODE = "fr-eco-friendly-mode";
+    public static final String CHANNEL_RE_WATER_FILTER = "fr-water-filter";
```

Comment:
Some door, because it's side-by-side refrigerators (2 doors). The refrigerator warn if some door is opened, but doesn't distingue which one is, or both :-(. I now "some-door" sounds strange but if you have some tip...
About the freezer / fridge / fr, I will sanitize it :-D.

Reply:
In that case it makes more sense. Just `door-open` might be sufficient